src/shared/utils/profileUtils.ts

```diff
@@ -0,0 +1,28 @@
+export const getRandomColor = (string: string) => {
```

Comment:
This is something I built a while back to get a randomColor that looks a bit cleaner
`export const getRandomColor = () => "#" + Math.floor(Math.random() * 16777215).toString(16)`